styles/components/card.css

```diff
@@ -0,0 +1,36 @@
+.card {
+  background-color: var(--color-blue);
+  position: relative;
+  border: 2px solid black;
+  border-radius: 1em;
+  padding: 0.5em;
+  text-align: center; 
+  box-shadow: 0.5rem 0.5rem 0.5rem black;
+  max-width: 25rem;
+  height: auto;
+}
+.card__bookmark {
+  position: absolute;
+  right: 20px;
+  top: -10px;
+  border-style: none;
+  background-color: transparent;
+  z-index: 1;
+}
+.card__show-answer {
```

Comment:
Aren´t we supposed to use two dashes (--) when giving a class a modifier? For example `<card__show--answer>`

Reply:
I think that isn't a modifier. But im not sure... 😅